### DISCUSSION TITLE
What is the decomposition method of NeuralProphet？

### DISCUSSION THREAD

**Discussion by mmyyyzpl** [ourownstory/neural_prophet#1419] [Q&A - get help using NeuralProphet]
Hello, I would like to inquire about the method of decomposing time series in the neuralprophet model and how to decompose the time series into trend component, seasonal components, autoregressive component, etc？

Perhaps it's because I'm not familiar with the underlying code of the model or the paper, and I don't seem to have found relevant content

**Comment by SimonWittner** [comment]
Hi,
the forecasts in NeuralProphet are additive. So you could add up the components to get the forecast. I recommend you to have a look at https://neuralprophet.com/tutorials/tutorial03.html. There you can find an example with two components (trend and seasonality).
